ote/src/cljs/ote/views/transit_visualization.cljs

```diff
@@ -723,8 +734,8 @@
 
          (when (and hash->color date->hash)
            [:span
+            ;; Select first trip to show stops
```

Comment:
I don't understand the above comment, what does it reference... should it be in a controller event?